internal/gps/pkgtree/reachmap.go

```diff
@@ -20,44 +20,37 @@ type ReachMap map[string]struct {
 	Internal, External []string
 }
 
-// FlattenAll flattens a reachmap into a sorted, deduplicated list of all the
-// external imports named by its contained packages.
-//
-// If stdlib is false, then stdlib imports are excluded from the result.
-func (rm ReachMap) FlattenAll(stdlib bool) []string {
```

Comment:
Heh, I think that might bring us back around to something more similar to the API like two iterations ago 😛 

I'm gonna backtrack and just say it's fine to drop this method. I don't have the bandwidth to quibble about things we're not actually using 😄